kubernetes/linux/setup.sh

```diff
@@ -1,9 +1,27 @@
-TMPDIR="/opt"
-cd $TMPDIR
+cd /opt
+
+# set up apt repo for ruby2.6
+sudo apt-get install software-properties-common -y
+sudo apt-add-repository ppa:brightbox/ruby-ng -y
+# set up apt repo for fluent-bit(td-agent-bit)
+wget -qO - https://packages.fluentbit.io/fluentbit.key | sudo apt-key add -
+sudo echo "deb https://packages.fluentbit.io/ubuntu/xenial xenial main" >> /etc/apt/sources.list
+
+sudo apt-get update
+
+apt-get install -y libc-bin wget openssl curl sudo python-ctypes init-system-helpers  net-tools rsyslog cron vim dmidecode apt-transport-https gnupg
+
+# install telegraf in parallel with everything else to save build time
+(
+    wget https://dl.influxdata.com/telegraf/releases/telegraf-1.18.0_linux_amd64.tar.gz
+    tar -zxvf telegraf-1.18.0_linux_amd64.tar.gz
+    mv /opt/telegraf-1.18.0/usr/bin/telegraf /opt/telegraf
+    chmod 777 /opt/telegraf
+) &
 
 #Download utf-8 encoding capability on the omsagent container.
 #upgrade apt to latest version
-apt-get update && apt-get install -y apt && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
+apt-get install -y apt && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
```

Comment:
how is this related to speedier builds ? also, we need to update the already existing apt package manager. Will install on top existing update to latest version as well ?

Reply:
I have no idea why we install apt. I didn't change that (the change on this line is removing `apt-get update`